training/README.md

```diff
@@ -57,6 +65,15 @@ Of course, the other Makefile variables are still available, so the following is
 make nvidia REGISTRY=myregistry.com REGISTRY_ORG=ai-training IMAGE_NAME=nvidia IMAGE_TAG=v1 FROM=registry.redhat.io/rhel9/rhel-bootc:9.4
 ```
 
+# How to build Cloud ready images
+
+Bootc container images can be installed on physical machines, virtual machines and in the cloud.  Often it is useful to add the cloud-init package when running the operarting systems in the cloud.
```

Comment:
I think we can shortly redirect to some upstream docs on this btw.

But the larger problem is that cloud-init isn't currently working for the growfs case so we should do https://github.com/containers/ai-lab-recipes/pull/410 first.